### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sbddesign**
Update readme for v1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,36 +2,44 @@
...
 
 You can verify that this worked by opening a shell and running:
 
-`dig txt satoshi.user._bitcoin-payment.twelve.cash`
+`dig txt stephen.user._bitcoin-payment.twelve.cash`
 
 The expected output should be:
 
-`satoshi.user._bitcoin-payment.twelve.cash. 3600 IN TXT "bitcoin:?lno=lno1pgtyymmvwscnygzsv9uk6etwwssyzerywfjhxuckyypme4su43qu44v09r28p6whddr62zkyuj68ha68kky2za4zrdu9uuq"`
+`satoshi.user._bitcoin-payment.twelve.cash. 3600 IN TXT "bitcoin:?lno=lno1pgg8getnw3q8gam9d3mx2tnrv9eks93pqw7dv89vg89dtreg63cwn4mtg7js438yk3alw3a43zshdgsm0p08q"`
```
Comment (chdwlch) [inline README.md:42]: satoshi here `satoshi.user._bitcoin-payment`

Comment (sbddesign) [inline README.md:42]:
```suggestion
`stephen.user._bitcoin-payment.twelve.cash. 3600 IN TXT "bitcoin:?lno=lno1pgg8getnw3q8gam9d3mx2tnrv9eks93pqw7dv89vg89dtreg63cwn4mtg7js438yk3alw3a43zshdgsm0p08q"`
```

Comment (sbddesign) [reply README.md]: Fixed

**Review by chdwlch — APPROVED** [review]
👍 👍

**Pull request merged**